web/src/lib/components/user-settings-page/appearance-settings.svelte

```diff
@@ -105,6 +105,15 @@
         />
       </div>
 
+      <div class="ml-4">
+        <SettingSwitch
+          title="Play Video Thumbnail on Hover"
```

Comment:
Most of the titles in the user settings seem to have normal casing, the exceptions being Default Locale and now this setting. I think it's best to be consistent.

Reply:
Good point. Server settings is all uppercase. Some of user settings is all uppercase like under Account. I'll fix just my new item and do a separate PR to make the others consistent.